tests/unit/conftest.py

```diff
@@ -81,6 +80,57 @@ def tmppath(tmpdir: py.path.local) -> Path:
         return Path(str(tmpdir))
 
 
+@pytest.fixture
+def test_path() -> Path:
+    """Returns the path to the directory containing the tests."""
+    return Path(__file__).parent
+
+
+@pytest.fixture
+def make_path_fixture() -> Callable[[Path, Path, str], Generator[Path, None, None]]:
+    """
+    Provides a reusable helper function to create fixtures that populate a temporary directory.
+
+    The returned function creates a temporary directory, populates it with test
+    data from a given subdirectory, and changes the current working directory to it.
+    The original working directory is restored after the test.
+    """
+
+    def _helper(tmppath: Path, test_path: Path, name: str) -> Generator[Path, None, None]:
+        path = tmppath / name
+        shutil.copytree(test_path / name, path)
+
+        original_directory = Path.cwd()
+        os.chdir(path)
+
+        try:
+            yield path
+        finally:
+            os.chdir(original_directory)
+
+    return _helper
+
+
+@pytest.fixture
+def defined_path(
```

Comment:
I would reconsider whether these three fixtures are generic enough that any test could use them - `tmppath_factory` and `tmppath` are for sure, but `empty_path` and `defined_path`, looking at their code and corresponding unit tests, they look like fixtures more suited to be `test_id`-specific, i.e. defined in that file, with smaller scope.

Reply:
Moved `fixture_id_tree_empty` and `fixture_id_tree_defined` to test_utils.py . They are not used anywhere though.